### ISSUE TITLE
[gateway-gcs]: Support for bucket policies

### ISSUE THREAD

**Issue by krishnasrinivas** [minio/minio#4282]
Currently not supported:

```
// SetBucketPolicies - Set policy on bucket                                                                                                                                              
func (l *gcsGateway) SetBucketPolicies(bucket string, policyInfo policy.BucketAccessPolicy) error {                                                                                      
        return NotSupported{}                                                                                                                                                            
}                                                                                                                                                                                        
                                                                                                                                                                                         
// GetBucketPolicies - Get policy on bucket                                                                                                                                              
func (l *gcsGateway) GetBucketPolicies(bucket string) (policy.BucketAccessPolicy, error) {                                                                                               
        return policy.BucketAccessPolicy{}, NotSupported{}                                                                                                                               
}                                                                                                                                                                                        
                                                                                                                                                                                         
// DeleteBucketPolicies - Delete all policies on bucket                                                                                                                                  
func (l *gcsGateway) DeleteBucketPolicies(bucket string) error {                                                                                                                         
        return NotSupported{}                                                                                                                                                            
}
```

**Comment by nl5887** [comment]
Fixed in commit 2176b58a

**Comment by hackintoshrao** [comment]
Does this particular case/test exist in Mint ?
